Samples/Samples/ViewModel/ContactViewModel.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Windows.Input;
+using Xamarin.Essentials;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
```

Comment:
Shouldn't need this

Reply:
It's just to use the `ForEach` extension method. If you prefer I can use the normal one.

Reply:
Yeah, would just use the normal one

Reply:
done.